website/src/content/docs/docs/libraries/azure-resource-manager/reference/data-types.md

```diff
@@ -58,6 +75,15 @@ model Azure.ResourceManager.ArmAsyncOperationHeader<StatusMonitor, UrlValue, Fin
 | UrlValue      | The value type of the Azure-AsyncOperation header |
 | FinalResult   | The logical final result of the operation         |
 
+#### Examples
+
+```typespec
+op createOrUpdate is ArmResourceActionAsync<
```

Comment:
SHould not be createOrUpdate,  call this `alterWidget` and include WiddgetResource in the response parameter